templates/primary-secondary/primary-secondary.js

```diff
@@ -673,13 +675,8 @@ class TemplatePrimarySecondary extends RtlMixin(LocalizeCoreElement(LitElement))
 			:host([resizable]) [data-is-collapsed] aside {
 				visibility: hidden;
 			}
-			:host([resizable]:not([dir="rtl"]):not([secondary-first])) aside,
-			:host([resizable][dir="rtl"][secondary-first]) aside {
-				float: left;
-			}
-			:host([resizable][dir="rtl"]:not([secondary-first])) aside,
-			:host([resizable]:not([dir="rtl"])[secondary-first]) aside {
-				float: right;
+			:host([resizable]:not([secondary-first])) aside {
+				float: inline-start;
```

Comment:
In my testing, this `float` didn't seem necessary, but I am a little afraid to remove it. It think it is safe to convert to use flow-relative value given the original CSS.